.github/workflows/bootstrap/action.yml

```diff
@@ -14,7 +14,9 @@ runs:
     # Install .NET version as mandated by global.json
     - uses: actions/setup-dotnet@v4
       with:
-          global-json-file: global.json
```

Comment:
What's the reason for this change? Do we need both, net6 and net8 installed?

Reply:
This was needed for `nupkg-validator` but the bump to `0.5.0` should have fixed this 🤞 will revert this to verify. 

Starting with .NET 9.0 the SDK check on dotnet tools is relaxed which is great.